types/meteor/globals/accounts-base.d.ts

```diff
@@ -146,3 +146,23 @@ declare module Accounts {
 declare module Accounts {
     function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
 }
+
+declare module Accounts {
```

Comment:
I chose to continue with the existing style of adding a new module for every change